src/core/state/backendNetworks/backendNetworks.ts

```diff
@@ -0,0 +1,595 @@
+        );
+
+        return (
+          (chainId ? chainsGasUnits[chainId] : undefined) ||
+          chainsGasUnits[ChainId.mainnet]
+        );
+      },
+    ),
+
+    getChainDefaultRpc: createParameterizedSelector(() => (chainId) => {
+      const defaultChains = get().getDefaultChains();
+      return defaultChains[chainId].rpcUrls.default.http[0];
+    }),
+  }),
+
+  {
+    partialize: (state) => ({
+      backendChains: state.backendChains,
+      backendNetworks: state.backendNetworks,
+    }),
+    storageKey: 'backendNetworks',
+  },
+);
+
+/** -----------------------------------------------------------------------------------
+ *  Backend networks helper functions.
+ *  Some of these defaults, e.g. gas speeds, should eventually come from the backend.
+ * ------------------------------------------------------------------------------------*/
+
+function toChainId(id: string): ChainId {
+  return parseInt(id, 10);
+}
+
+function getDefaultGasSpeeds(chainId: ChainId): GasSpeed[] {
+  switch (chainId) {
+    case ChainId.bsc:
+    case ChainId.goerli:
+    case ChainId.polygon:
+      return [GasSpeed.NORMAL, GasSpeed.FAST, GasSpeed.URGENT];
+    case ChainId.gnosis:
+      return [GasSpeed.NORMAL];
+    default:
+      return [GasSpeed.NORMAL, GasSpeed.FAST, GasSpeed.URGENT, GasSpeed.CUSTOM];
+  }
+}
+
+function getDefaultPollingInterval(chainId: ChainId): number {
+  switch (chainId) {
+    case ChainId.polygon:
+      return 2_000;
+    case ChainId.arbitrum:
+    case ChainId.bsc:
+      return 3_000;
+    default:
+      return 5_000;
+  }
+}
+
+function getDefaultSimplehashNetwork(chainId: ChainId): string {
+  switch (chainId) {
+    case ChainId.apechain:
+      return 'apechain';
+    case ChainId.arbitrum:
+      return 'arbitrum';
+    case ChainId.avalanche:
+      return 'avalanche';
+    case ChainId.base:
+      return 'base';
+    case ChainId.blast:
+      return 'blast';
+    case ChainId.bsc:
+      return 'bsc';
+    case ChainId.degen:
+      return 'degen';
+    case ChainId.gnosis:
+      return 'gnosis';
+    case ChainId.goerli:
+      return 'ethereum-goerli';
+
+    // FIXME: Unsupported as of now https://docs.simplehash.com/reference/supported-chains-testnets#mainnets
+    // case ChainId.gravity:
+    //   return 'gravity';
+
+    // FIXME: Unsupported as of now https://docs.simplehash.com/reference/supported-chains-testnets#mainnets
+    // case ChainId.ink:
+    //   return 'ink';
+
+    case ChainId.mainnet:
+      return 'ethereum';
+    case ChainId.optimism:
+      return 'optimism';
+    case ChainId.polygon:
+      return 'polygon';
+
+    // FIXME: Unsupported as of now https://docs.simplehash.com/reference/supported-chains-testnets#mainnets
+    // case ChainId.sanko:
+    //   return 'sanko';
+
+    case ChainId.scroll:
+      return 'scroll';
+    case ChainId.zksync:
+      return 'zksync-era';
+    case ChainId.zora:
+      return 'zora';
+    case ChainId.linea:
+      return 'linea';
```

Comment:
In the [NFT query](https://github.com/rainbow-me/browser-extension/blob/af6b2daf5b88a8bc478dd7a6f66405a5a4e3d2b3/src/core/resources/nfts/galleryNfts.ts#L55) we fetch all NFTs for active networks and `gnosis` (for POAPs), even when they are not officially supported networks. That's the main reason we have had to maintain chain naming mappings (to stay in sync with SimpleHash) and append to `ChainName` and `ChainId` types. Would love to just get this as a value from backend so we can blow away all of that logic and types. We also don't use the `nftProxy` right now because Chris did work to optimize pagination for direct fetching from SimpleHash; that's probably why this data need diverged from App.